### DISCUSSION TITLE
Increasing Number of Albums Shown Per Page

### DISCUSSION THREAD

**Discussion by maxycatz** [jcorporation/myMPD#1363] [Q&A] [answered]
Switch pages while browsing or scrolling through library breaks the flow for me. 

What can be changed to enable the number of albums shown go beyond 500?

Cheers

**Comment by jcorporation** [comment]
I can add an option for 1000 Elements.

**Comment by maxycatz** [comment]
I added the value 1000 option in:
../htdocs/js/globales.js
../htdocs/js/settings.js
..//htdocs/js/pagination.js
And it's working fine.

But maybe alternatives could be: increasing the available number values, an '0'  for unlimited, or user input.

Thanks for this brilliant piece of work; myMPD is by far the best MPD client I have used ( having recently explored probably all other current Album art focused ones!)

**Comment by jcorporation** [comment, marked as answer]
I increased it to 1000. In the past there was an option to set it to unlimited, but this caused issues for large databases.

Commit: 25d66e9a3f8d12a3e156d6de5771340957541b3b
